packages/select-rich/src/LionSelectRich.js

```diff
@@ -112,6 +116,7 @@ export class LionSelectRich extends SlotMixin(ScopedElementsMixin(OverlayMixin(L
     this.interactionMode = 'auto';
 
     this.singleOption = false;
+    this.arrowWidth = 28;
```

Comment:
The arrow is 15.x px wide, so let's say 16. And then you have to account for a difference of 4px in padding between content and invoker nodes, which amounts to 8px (both sides). I checked, and an arrowWidth of 24px is indeed enough spacing, but 28px definitely looks nicer... 

Seems a little weird, I don't think we should overcomplicate it by calculating padding diffs :P so I think I'm alright with the subclasser just setting a proper arrow width.

Should we make it a protected property though? Since it's not really meant to be used by the app developer, but it should be set by the subclasser?

Reply:
I tried to get the padding.. but was not working (can't read from :host selector).

I'm fine with no property.